### ISSUE TITLE
String column specification produces format 'text' instead of 'raw'

### ISSUE THREAD

**Issue by klimov-paul** [yii2tech/csv-grid#18]
String column specification produces format 'text' instead of 'raw'.
For example:

```php
$exporter = new CsvGrid([
    ///...,
    'columns' => [
        ['attribute' => 'name'], // format detected as 'raw'
        'description', // format detected as 'text'
    ],
]);
```

**Comment by klimov-paul** [comment]
Resolved by commit 09ee839a9a129ffe1ede0fecdbe839a3678542b5

**Comment by GuillaumeAllegre** [comment]
thanks !
